packages/e2e-tests/cypress/integration/eligibility/householder-planning/date-decision-due/date-decision-due-steps.js

```diff
@@ -12,6 +12,7 @@ import { goToAppealsPage } from '../../../../support/common/go-to-page/goToAppea
 import { selectPlanningApplicationType } from '../../../../support/eligibility/planning-application-type/select-planning-application-type';
 import { selectSiteOption } from '../../../../support/eligibility/appellant-selects-the-site/select-site-option';
 import { selectPlanningApplicationDecision } from '../../../../support/eligibility/granted-or-refused-application/select-planning-application-decision';
+import {clickContinueButton} from "../../../../support/common/clickContinueButton";
 import {
   getDateDecisionDueDay,
   getPlanningApplicationDecisionError,
```

Comment:
Does anything need to happen with the commented out lines 120 & 122?

Reply:
@demzzydev Can you please un comment them now that we can check the date format?